tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -1425,8 +1425,9 @@ public function testNotEquals()
     public function testSameAs()
     {
         $validator = new Validator();
-        $this->assertProxyMethod($validator, 'sameAs', 'other', ['other', true], 'compareFields');
+        $this->assertProxyMethod($validator, 'sameAs', 'other', ['other', '==='], 'compareFields');
```

Comment:
You chould use the const also for '===' etc.

Reply:
There is one `Validation::SAME_AS` but not used in tests.

Reply:
Yeah, I mean you could reuse the const existing in the Validation class here to be more speaking and to avoid typos.